server/config.ts

```diff
@@ -98,14 +98,6 @@ export default {
       },
       agent: new AgentConfig(),
     },
-    ciagApi: {
```

Comment:
I've changed the relevant client classes to use the properties from the `educationAndWorkPlanApi` object, rather than the `ciagApi` object.
(They pointed to the same thing after we migrated the original CIAG API to the PLP API)